### DISCUSSION TITLE
when a mousewheel click, open in a new tab

### DISCUSSION THREAD

**Discussion by paubou** [noah-nuebling/mac-mouse-fix#1025] [Questions]
Hello,
One of the default mouse shortcut I liked was the ability to mousewheel click on a link to open it in a new tab. Since I installed the app, this is not possible anymore. Am I missing a setting or something else? 
Thanks!

**Comment by user68329** [comment]
Yea this feature broke for me too - middle button no longer works. It may have broke with one of the macOS updates, maybe 6+ months ago.

**Comment by noah-nuebling** [comment]
Hey guys, try remapping to the 'Middle Click' action, it does that!
